website/server/controllers/api-v3/members.js

```diff
@@ -33,7 +33,7 @@ let api = {};
 api.getMember = {
   method: 'GET',
   url: '/members/:memberId',
-  middlewares: [authWithHeaders()],
+  middlewares: [],
```

Comment:
This is fine. It's what is supposed to be used for the front page when sharing the member modal.